### PR TITLE
RGRIDT-908: SetColumnAggregate client action

### PR DESCRIPTION
**Pull request by glundgren93**
This PR delivers the necessary changes to deliver SetColumnAggregate client action. This action will set the aggregate function on columns inside the group panel.

### What was done
* Created SetColumnAggregate API method.
* Created new method on GroupPanel that sets aggregate into the function

### Test Steps
1. Open Sample
2. Group description column
3. Price column should have the total sum on groups.

### Checklist
* [x] tested locally
* [x] documented the code
* [x] clean all warnings and errors of eslint
* [x] requires changes in OutSystems (if so, provide a module with changes)
* [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

**Review by tmbp95 — DISMISSED** [review]

File: code/src/GridAPI/ColumnManager.ts [hunk 1]
```diff
@@ -163,4 +163,32 @@ namespace GridAPI.ColumnManager {
...
+     *
+     * @export
+     * @param {string} gridID ID of the Grid where the change will occur.
+     * @param {string} columnID id of the column with which actions on the column can be performed.
+     * @param {wijmo.Aggregate} aggregate aggregate that will be applied on group panel.
+     */
+    export function SetColumnAggregate(
+        gridID: string,
+        columnID: string,
+        aggregate: wijmo.Aggregate
```
Comment (rugoncalves) [inline code/src/GridAPI/ColumnManager.ts:178]: Why are we passing an wijmo as an input to the API?

**Review by rugoncalves — CHANGES_REQUESTED** [review]

File: code/src/GridAPI/ColumnManager.ts (hunk 1 above)
Comment (tmbp95) [reply code/src/GridAPI/ColumnManager.ts:178]: opa, nice catch!

Comment (glundgren93) [reply code/src/GridAPI/ColumnManager.ts]: fixed

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

File: code/src/WijmoProvider/Features/GroupPanel.ts [hunk 2]
```diff
@@ -121,6 +121,14 @@ namespace WijmoProvider.Feature {
             );
         }
 
+        public setAggregate(binding: string, aggregate: wijmo.Aggregate): void {
+            const column = this._grid.getColumn(binding);
+
+            if (column) {
+                column.provider.aggregate = wijmo.Aggregate[aggregate];
```
Comment (tmbp95) [inline code/src/WijmoProvider/Features/GroupPanel.ts:128]: what happens if the aggregate is not valid?

**Review by tmbp95 — APPROVED** [review]

**Review by rugoncalves — CHANGES_REQUESTED** [review]

**Review by rugoncalves — APPROVED** [review]

**Review by tmbp95 — APPROVED** [review]

**Pull request merged**